packages/ui/src/components/templates/report/components/AdsAndSocialMedia/AdsAndSocialMedia.tsx

```
@@ -1,104 +1,236 @@
+        })}
+      </div>
+    </div>
+
+    {/* TODO: Shows failing state by default for now */}
+    <div>
+      <h3 className="mb-2 text-base font-bold">Ads</h3>
+      <Card
+        className={ctw(
+          'flex w-full justify-between p-4 shadow-lg',
+          // relatedAdsImages && relatedAdsImages.length > 0 ? 'opacity-100' : 'opacity-60',
+          false ? 'opacity-100' : 'opacity-60',
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)